daemon/config/config_unix.go

```diff
@@ -1,10 +1,12 @@
-// +build linux freebsd
+// +build !windows
```

Comment:
Can we rename the file to _linux.go and remove the build tag?

I don’t think the code in this file works on non-Linux.

Reply:
Yeah, makes sense; I thought "if it compiles, it compiles" but probably better to be explicit (as we for sure don't _test_, or _assume_ other unix-y environments for these)

Updated 👍